manifests/charts/istio-control/istio-discovery/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: istio-reader-clusterrolebinding{{- if not (eq .Values.revision "")}}-{{ .Values.revision }}{{- end }}-{{ .Values.global.istioNamespace }}
```

Comment:
Since we are moving things around and creating new resources - can we drop 'clusterrolebinding' 'clusterrole' 'service-account' suffix from the name ? It is already in the Kind...

Reply:
Fixed